### PR TITLE
Hello example

### PR DESCRIPTION
**Pull request by nalaka**
Introductory example app demonstrating the use of JavaScript SDK
for Cognite Fusion with the Gearbox React components.

### CODE REVIEW

File: hello/src/Explorer.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+  };
+
+  handleAssetSelect(asset) {
+    this.setState({ selectedAssetID: asset.key });
+  }
+
+  render() {
+    return (
+      <div>
+        {/* <AssetDetailsPanel
```
[the remark this answers is not in the document]
Comment (nalaka) [reply hello/src/Explorer.js]: Thanks for the review- will fix this and update the pull request over the weekend.

**Comment by nalaka** [comment]
Hi @polomani & @f1cognite  - I removed that commented out code and extended the example. Also added a walkthrough to the README.

Regarding the CI check failure: looks like the build fails with an `openssl` error unrelated to this PR.

Please note that the branch is rebased on master.

**Comment by polomani** [comment]
@nalaka it fails because you are trying to merge from external repo.
I've added you as a collaborator, could you please open a PR from this repo branch instead?
And then I'll have a few suggestions how to makes the demo even better, so we can merge it.

**Comment by nalaka** [comment]
Hi @polomani, Created pull request #62.  

And, thanks for inviting me to be a collaborator!

**Pull request closed without merging**